chargeOclock/devTemp/devClientsViewController.swift

```diff
@@ -47,4 +51,32 @@ class devClientsViewController: UITableViewController {
 		}
 	}
 
+	func plusTapped() {
+
```

Comment:
this many empty lines in a function this small don't really improve readability 😉